osu.Server.Spectator.Tests/StatefulUserHubTest.cs

```diff
@@ -105,11 +104,12 @@ public async Task SameUserOldConnectionDoesntDestroyNewState()
 
         private class TestStatefulHub : StatefulUserHub<object, ClientState>
         {
-            public EntityStore<ClientState> UserStore => ACTIVE_STATES;
+            public EntityStore<ClientState> ActiveUsers { get; }
```

Comment:
Not sure why this rename has been shoved into this diff, inflating it significantly. It seems mostly unnecessary.

Reply:
It's just a name change in the end to match the entity store name, but I could revert that I guess.